ibm_was/datadog_checks/ibm_was/ibm_was.py

```diff
@@ -0,0 +1,127 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import requests
+from lxml import etree
+
+from six import iteritems
+
+from datadog_checks.base import AgentCheck, ensure_unicode, is_affirmative
+
+from . import metrics, validation
+
+
+class IbmWasCheck(AgentCheck):
+
+    SERVICE_CHECK_CONNECT = "ibm_was.can_connect"
+    METRIC_PREFIX = 'ibm_was'
+
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        AgentCheck.__init__(self, name, init_config, agentConfig, instances)
+        self.metric_type_mapping = {
+            'AverageStatistic': self.gauge,
+            'BoundedRangeStatistic': self.gauge,
+            'CountStatistic': self.monotonic_count,
+            'DoubleStatistic': self.rate,
```

Comment:
what is a double and why is it a rate?

Reply:
We had a mini discussion about it here - https://github.com/DataDog/integrations-core/pull/2846#discussion_r244946822 Mainly because of what the metric types are defined as in the metrics.py file (link to the IBM docs is there too)